test/mesh.jl

```diff
@@ -114,6 +114,15 @@
       @test_reference "data/grid-2D-$T-data.png" plot(CartesianGrid{T}(10,10),1:100)
       # @test_reference "data/grid3D-data.png" plot(RegularGrid(10,10,10),1:1000)
     end
+
```

Comment:
Please move these tests to before the visual tests section. We put visual tests at the end because they are usually more problematic to pass when plotting packages get updated.